src/test/java/org/folio/search/controller/BrowseCallNumberConsortiumIT.java

```diff
@@ -0,0 +1,169 @@
+import static org.awaitility.Durations.TWO_HUNDRED_MILLISECONDS;
+import static org.awaitility.Durations.TWO_MINUTES;
+import static org.folio.search.support.base.ApiEndpoints.instanceSearchPath;
+import static org.folio.search.support.base.ApiEndpoints.recordFacetsPath;
+import static org.folio.search.utils.SearchUtils.getIndexName;
+import static org.folio.search.utils.TestConstants.CENTRAL_TENANT_ID;
+import static org.folio.search.utils.TestConstants.FOLIO_CN_TYPE;
+import static org.folio.search.utils.TestConstants.LOCAL_CN_TYPE;
+import static org.folio.search.utils.TestConstants.MEMBER_TENANT_ID;
+import static org.folio.search.utils.TestUtils.array;
+import static org.folio.search.utils.TestUtils.facet;
+import static org.folio.search.utils.TestUtils.facetItem;
+import static org.folio.search.utils.TestUtils.mapOf;
+import static org.folio.search.utils.TestUtils.parseResponse;
+import static org.folio.search.utils.TestUtils.randomId;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+import static org.opensearch.index.query.QueryBuilders.matchAllQuery;
+import static org.opensearch.search.builder.SearchSourceBuilder.searchSource;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Stream;
+import org.assertj.core.util.Lists;
+import org.folio.search.domain.dto.Facet;
+import org.folio.search.domain.dto.FacetResult;
+import org.folio.search.domain.dto.Holding;
+import org.folio.search.domain.dto.Instance;
+import org.folio.search.domain.dto.Item;
+import org.folio.search.domain.dto.ItemEffectiveCallNumberComponents;
+import org.folio.search.domain.dto.RecordType;
+import org.folio.search.support.base.BaseConsortiumIntegrationTest;
+import org.folio.search.utils.SearchUtils;
+import org.folio.spring.testing.type.IntegrationTest;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.opensearch.action.search.SearchRequest;
+import org.opensearch.client.RequestOptions;
+import org.opensearch.client.RestHighLevelClient;
+import org.springframework.beans.factory.annotation.Autowired;
+
+@IntegrationTest
+class BrowseCallNumberConsortiumIT extends BaseConsortiumIntegrationTest {
+
+
+  private static final String[] LOCATIONS = array("location 1", "location 2");
+  private static final Instance[] INSTANCES_CENTRAL = instancesCentral();
+  private static final Instance[] INSTANCES_MEMBER = instancesMember();
+
+
+  @BeforeAll
+  static void prepare(@Autowired RestHighLevelClient restHighLevelClient) throws InterruptedException {
+    setUpTenant(CENTRAL_TENANT_ID, INSTANCES_CENTRAL);
+    setUpTenant(MEMBER_TENANT_ID);
+    saveRecords(MEMBER_TENANT_ID, instanceSearchPath(), asList(INSTANCES_MEMBER),
+      INSTANCES_CENTRAL.length,
```

Comment:
maybe you wanted to put here: ```INSTANCES_MEMBER``` or I did not get the idea - saving member's instances but expect 2 events being received ?

Reply:
also, why in a next line to save the member's instances again?

Reply:
Yes, typo, changed to 4 expected instances.
Copied from some old local test class, didn't notice, thanks, removed